src/script/cpp_api/s_security.cpp

```diff
@@ -244,6 +244,7 @@ void ScriptApiSecurity::initializeSecurityClient()
 		"rawset",
 		"select",
 		"setfenv",
+		"getmetatable",
```

Comment:
getmetatable can be used to escape the sandbox

Reply:
and it breaks dump()

Reply:
Might not be an issue with CSM since the insecure environment shouldn't exist in the Lua state?

Reply:
What's not an issue? The line is reverted to keep the CSM env more or less secure.